### PR TITLE
percona-xtrabackup: 2.4.9 -> 2.4.12

### PR DESCRIPTION
**Pull request by Izorkin**
###### Motivation for this change
Update version

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build percona-xtrabackup

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -4364,6 +4364,7 @@ with pkgs;
   pepper = callPackage ../tools/admin/salt/pepper { };
 
   percona-xtrabackup = callPackage ../tools/backup/percona-xtrabackup {
+    stdenv = overrideCC stdenv gcc5;
```
Comment (xeji) [inline pkgs/top-level/all-packages.nix]: Do we really need to downgrade to gcc5? Fixing the build with gcc7 would be the preferred solution.

Comment (Izorkin) [reply pkgs/top-level/all-packages.nix]: I do not know how to fix an error when building with gcc7

**Comment by xeji** [comment]
This builds on my machine with gcc7 but I have not tested the resulting app.
@Izorkin, please test.

**Comment by Izorkin** [comment]
@xeji thanks. Works.

**Comment by xeji** [comment]
@GrahamcOfBorg build percona-xtrabackup

**Pull request merged**